Packs/CiscoSpark/Integrations/CiscoWebexEventCollector/CiscoWebexEventCollector_description.md

```diff
@@ -43,6 +43,12 @@ In order to receive all events, You must run all three commands twice, once with
 2. Run the ***cisco-webex-oauth-complete*** command with the **user** and **code** arguments The **user** argument should be set to the same value as in the previous command (`admin` or `compliance officer`). The **code** argument should be set to the value returned in the code query parameter from the previous command.
 3. Run the ***cisco-webex-oauth-test*** command with the **user** argument. The **user** argument should be set to the same value as in the previous command (`admin` or `compliance officer`) to ensure connectivity to Cisco Webex.
 
-### Fetch security audits
-This API requires Full Admin Permission.
-In order to fetch security audit events, pro pack needs to be installed on the Webex instance.
+### Fetch Security Audits  
+
+This API requires Full Admin permissions.  
+
+To fetch security audit events, the **Pro Pack** must be installed on the Webex instance. Additionally, the **Allow user authentication data** setting must be enabled:  
```

Comment:
Lets make sure it is also in the readme.

Reply:
Done